content/locales/fr.ts

```diff
@@ -51,4 +51,26 @@ export const fr = {
   "/account/locale": {
     title: "Changer de langue",
   },
+  alertMessages: {
+    localeChanged: "Votre langue a été réglée sur",
+    confirmationCodeEmail: "Un e-mail de confirmation a été envoyé",
+    confirmationCodePhone: "Un SMS de confirmation a été envoyé",
+    newConfirmationCodeEmail: "Un nouvel e-mail de confirmation a été envoyé",
+    newConfirmationCodePhone: "Un nouveau SMS de confirmation a été envoyé",
+    validationCodeExpired: "Le code de validation a expiré",
+    emailUpdated: "Votre address e-mail a été mise à jour",
+    phoneUpdated: "Votre numéro de téléphone a été mis à jour",
+    emailAdded: "Votre adresse e-mail a été ajoutée",
+    phoneAdded: "Votre numéro de téléphone a été ajouté",
+    emailDeleted: "Votre adresse e-mail a été supprimée",
+    phoneDeleted: "Votre numéro de téléphone a été supprimé",
+    emailMarkedAsPrimary: "est maintenant votre e-mail principal",
+    phoneMarkedAsPrimary: "est maintenant votre numéro de téléphone principal",
+    addressAdded: "Votre adresse a été ajoutée",
+    addressUpdated: "Votre adresse a été mise à jour",
+    addressDeleted: "Votre adresse a été supprimée",
+    addressMarkedAsPrimary: "Votre adresse a été marquée comme principale",
+    profileCreated: "Your profile has been created",
+    profileUpdated: "Your profile has been updated",
```

Comment:
These keys are not present in english file, and are not translated in french.

Reply:
Damn nice catch.. I guess I was too quick in doing this. I checked some alert message flows, but not the ones regarding the profile ;/